src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,57 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        List<Driver> driversList = new ArrayList<>();
+        driversList.add(new Driver("Oleksii", "AST 445823"));
```

Comment:
you can initialize it by using `        Driver volodymyr = driverService.create(new Driver("Oleksii", "AST 445823"));
                                                       driverService.create(Oleksii);
` but i'm not sure if my version is better, let's mentor decide 😃